torch/_inductor/config.py

```diff
@@ -185,11 +185,20 @@ def is_fbcode():
     "requires_optimizer": True,
 }
 
-# for pattern torch.mm(a, b.to(dtype)) with cuda tensors, always use
-# torch._inductor.kernel.mm.tuned_mixed_mm's fused kernel.
-# Autotune will not compare with normal cast->then->mm option.
-# (if force_mixed_mm is true, the use_mixed_mm flag will be ignored)
-force_mixed_mm = False
+# force_mixed_mm can be used to control the behaviour for pattern torch.mm(a, b.to(dtype)) with cuda tensors.
+# The fallback aten implementation is normal cast->then->mm option.
+# If force_mixed_mm is "no": this flag will be ignored.
+# If forced_mixed_mm is "triton":
+# - Always use torch._inductor.kernel.mm.tuned_mixed_mm's fused kernel.
+# - Autotune will not compare with fallback.
+# If force_mixed_mm is "aten": always use the fallback aten implementation.
+# If force_mixed_mm is "heuristic":
+# - Enables the heuristic.
+# - If the heuristic decides to add a config, it will add the config as the first choice.
+# - If autotune is disabled, this config will always be chosen.
+# - If autotune is enabled, it will also compare with fallback aten implementation and fused kernel.
+# The use_mixed_mm flag will be ignored if forced_mixed_mm != "no".
+force_mixed_mm = "no"
```

Comment:
I will do a perf run to see how setting `force_mixed_mm = "heuristic"` and `use_mixed_mm = True` affects performance.

The behavior of setting `force_mixed_mm = "off"` should be equivalent to what was previously `force_mixed_mm = False`. If I understand correctly, it will only do autotuning if one of the autotuning flags is enabled? Otherwise, it will just use fallback if `force_mixed_mm = "off"` and `use_mixed_mm = True`.